operator-framework-core/pom.xml

```diff
@@ -47,6 +47,19 @@
           <commitIdGenerationMode>full</commitIdGenerationMode>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
```

Comment:
Why is this needed?

Reply:
To fill in the information that is available in Maven.